packages/core/js-client/src/services/__test__/builtInHandler.spec.ts

```diff
@@ -51,32 +54,32 @@ describe("Tests for default handler", () => {
     serviceId  | fnName               | args                                       | retCode | result
     ${"op"}    | ${"identity"}        | ${[]}                                      | ${0}    | ${{}}
     ${"op"}    | ${"identity"}        | ${[1]}                                     | ${0}    | ${1}
-    ${"op"}    | ${"identity"}        | ${[1, 2]}                                  | ${1}    | ${"identity accepts up to 1 arguments, received 2 arguments"}
+    ${"op"}    | ${"identity"}        | ${[1, 2]}                                  | ${1}    | ${"Expected 1 argument(s). Got 2"}
     ${"op"}    | ${"noop"}            | ${[1, 2]}                                  | ${0}    | ${{}}
     ${"op"}    | ${"array"}           | ${[1, 2, 3]}                               | ${0}    | ${[1, 2, 3]}
     ${"op"}    | ${"array_length"}    | ${[[1, 2, 3]]}                             | ${0}    | ${3}
-    ${"op"}    | ${"array_length"}    | ${[]}                                      | ${1}    | ${"array_length accepts exactly one argument, found: 0"}
+    ${"op"}    | ${"array_length"}    | ${[]}                                      | ${1}    | ${"Expected 1 argument(s). Got 0"}
```

Comment:
If these errors happen, can you as a developer understand that error happened in e.g. array_length function? Is there a stack trace? If for developers it's possbile to understand this then probably these errors are ok. If not - then previous erros where deffinetly better